public/blog/2025-01-bio-foundation-models/2025-01-bio-fountation-models.md

```diff
@@ -0,0 +1,84 @@
+---
+author: "Martin Špendl"
+date: "2025-01-10"
+draft: false
+title: "Foundation models in Orange with a just few lines of code"
+thumbImage: "ESM2-thumb.png"
+frontPageImage: "ESM2-thumb.png"
+blog: ["analysis", "bioinformatics", "Foundation Models"]
+shortExcerpt: "Foundation models can be used in Orange in just a few lines of code. Python Script can unlock every model from Hugging Face, if you are brave enough."
+longExcerpt: "Hugging Face foundation models can be used in Orange. With a few lines of code in the Python Script widget, you can download and run any model from Hugging Face repository and unlock new levels of analysis in your Workflows."
+---
+
+Foundation models are becoming a necessary addition to our toolbox, especially in fields like biomedicine. A large community of researchers is developing these models, so you don’t have to! Improved versions of models are uploaded almost by the minute on repositories like Hugging Face. The following blog/tutorial will show you how to use models from Hugging Face in Orange with just a bit of coding. Fear not; you will be only required to write a few lines in the Python Script widget, and you will unlock the full power of foundation models. Let’s begin!
+
+## What You’ll Need
+
+Before we get started, make sure you’ve got Orange installed on your computer. You’ll also need Python packages like torch and transformers. To install these packages, open Orange, navigate to Options and select Add-ons. In the upper right corner, click _Add more…_ and type torch. Once you can see the package in the list of add-ons, check the square and press OK on the bottom left corner of the window. This procedure will install the package torch  in your Orange application. Use the same procedure for the transformers package. You will only have to do this once. You can quickly check if your installation was successful by opening the Python Script widget, typing in the console: import transformers, and pressing Enter. If there is no error, the installation was successful. 
```

Comment:
Package names should stand out somehow, because now, if someone reads sloppily, they could install only one.